templates/default.mustache

```diff
@@ -531,29 +531,50 @@ endif
 
 " LSP highlighting
 if has("nvim")
-  hi default link LspDiagnosticsSignError    ErrorSign
-  hi default link LspDiagnosticsSignWarning  WarningSign
-  hi default link LspDiagnosticsSignInfo     InfoSign
-  hi default link LspDiagnosticsSignHint     HintSign
-
-  hi default link LspDiagnosticsVirtualTextError    ErrorSign
-  hi default link LspDiagnosticsVirtualTextWarning  WarningSign
-  hi default link LspDiagnosticsVirtualTextInfo     InfoSign
-  hi default link LspDiagnosticsVirtualTextHint     HintSign
-
-  hi default link LspDiagnosticsFloatingError    ErrorFloat
-  hi default link LspDiagnosticsFloatingWarning  WarningFloat
-  hi default link LspDiagnosticsFloatingInfo     InfoFloat
-  hi default link LspDiagnosticsFloatingHint     HintFloat
-
-  hi default link LspDiagnosticsUnderlineError    ErrorHighlight
-  hi default link LspDiagnosticsUnderlineWarning  WarningHighlight
-  hi default link LspDiagnosticsUnderlineInfo     InfoHighlight
-  hi default link LspDiagnosticsUnderlineHint     HintHighlight
-
-  hi default link LspReferenceText   ReferenceText
-  hi default link LspReferenceRead   ReferenceRead
-  hi default link LspReferenceWrite  ReferenceWrite
+  hi! link DiagnosticError  ErrorSign
+  hi! link DiagnosticWarn   WarningSign
+  hi! link DiagnosticInfo   InfoSign
+  hi! link DiagnosticHint   HintSign
+
+  hi! link DiagnosticFloatingError  ErrorFloat
+  hi! link DiagnosticFloatingWarn   WarningFloat
+  hi! link DiagnosticFloatingInfo   InfoFloat
+  hi! link DiagnosticFloatingHint   HintFloat
+
+  hi! link DiagnosticUnderlineError  ErrorHighlight
+  hi! link DiagnosticUnderlineWarn   WarningHighlight
+  hi! link DiagnosticUnderlineInfo   InfoHighlight
+  hi! link DiagnosticUnderlineHint   HintHighlight
+
+  hi! link DiagnosticsVirtualTextError    ErrorSign
+  hi! link DiagnosticsVirtualTextWarning  WarningSign
+  hi! link DiagnosticsVirtualTextInfo     InfoSign
+  hi! link DiagnosticsVirtualTextHint     HintSign
+
+  " Remove untill endif on next nvim release
```

Comment:
Provide more details?

- Which Neovim release exactly (I could be using nightly)
- Add an explanation as to why this can be removed later